lumigator/python/mzai/backend/backend/tests/api/routes/test_api_workflows.py

```diff
@@ -0,0 +1,51 @@
+from time import sleep
+
+import pytest
+import requests
+from fastapi.testclient import TestClient
+from schemas.datasets import DatasetFormat, DatasetResponse
+
+
+# First test waits and polls up to 10 times for the real backend to be up.
+def test_health_ok(local_client: TestClient):
```

Comment:
we can use FastAPI decorated idioms here as well, `on_startup`

https://fastapi.tiangolo.com/advanced/testing-events/

so we should be able to do something like,

```python
@app.on_event("startup")
def test_health_ok(local_client: TestClient):
        response = local_client.get("/health/")
        assert response.status_code == 200
```

Reply:
Oh, nice one, thanks!

Since the comment comes in after merging, I'm adding this to a small PR I'll open for a bugfix today/tomorrow.

Reply:
I included this in https://github.com/mozilla-ai/lumigator/pull/285 so no need 👍